static/styles/main.css

```diff
@@ -0,0 +1,170 @@
+/* ---------------------------------------------------------- GoogLe Fonts ---------------------------------------------------------- */
+@import url('https://fonts.googleapis.com/css2?family=Oleo+Script+Swash+Caps:wght@400;700&family=Poppins:ital,wght@0,300;0,500;1,600&display=swap');
+/* 
+font-family: 'Oleo Script Swash Caps', cursive;
+font-family: 'Poppins', sans-serif;
+ */
```

Comment:
Fetch with @OmerCS8 PR to remove the commented lines

Reply:
yes, Alon will fetch for #89 then I will for #94 and then for #96 :)

Reply:
![image](https://user-images.githubusercontent.com/36789773/163154795-ec9a7126-07fc-49f6-89a5-e8b7f83cbc9a.png)